include/MarketPlaceApp.h

```diff
@@ -429,6 +445,12 @@ class MarketPlaceApp : public PoolObjectSQL
     {
         return new MarketPlaceAppTemplate;
     }
+protected:
+
+    /**
+     * MarketPlaceApp format with no images
+     */
+    const std::set<std::string> no_image = {"ami"};
```

Comment:
I would propose not to depend on **ami** image format.
If this is the most straightforward way, let's use **external**, **remote**, or **cloud** keywords to identify no image to import.